src/reducer/product-reducer.js

```diff
@@ -0,0 +1,13 @@
+export default function productReducer(state, action) {
+  switch (action.type) {
+    case "ADD_PRODUCTS":
+      console.log("in reduce", [...state.products, ...action.payload]);
```

Comment:
You can remove the console.log statements, before merging.

Reply:
done! thanks for pointing this out